ui/src/main/java/io/streamzi/eventflow/API.java

```diff
@@ -159,17 +160,21 @@ public void postFlow(String flowJson) {
     @GET
     @Path("/topics")
     @Produces("application/json")
-    public List<String> listTopicNames() {
+    public String listTopicNames() throws Exception {
 
-        return container.getOSClient().customResources(
+        return MAPPER.writeValueAsString(container.getOSClient().customResources(
                 Crds.topic(),
                 KafkaTopic.class,
                 KafkaTopicList.class,
                 DoneableKafkaTopic.class)
                 .inNamespace(container.getOSClient().getNamespace()).list().getItems().stream()
-                .filter(kafkaTopic -> (kafkaTopic.getMetadata().getLabels().get("streamzi.io/source") == null))
-                .map(flow -> flow.getMetadata().getName())
-                .collect(Collectors.toList());
+                .filter(kafkaTopic -> (kafkaTopic.getMetadata().getLabels().get("streamzi.io/source") == null ))
+                .map(topic -> {
+                    String cloud = "local";
+                    return new TopicDetails(topic.getMetadata().getName(), cloud);
```

Comment:
I'd just put in `local` there directly, for now

Reply:
Good spot - fixed